pkg/crc/services/dns/dns.go

```diff
@@ -119,7 +119,7 @@ func CheckCRCPublicDNSReachable(serviceConfig services.ServicePostStartConfig) (
 		// try without using proxy
 		proxyConfig = &httpproxy.ProxyConfig{}
 	}
-	curlArgs := []string{"--head", publicDNSQueryURI}
+	curlArgs := []string{"--head", publicDNSQueryURI, "-L"}
```

Comment:
Follow is good, but I think it is better to do

`curl -sSL`

`-S`: This option is used in conjunction with `-s` (silent). It stands for "show error." When used with `-s`, it will still suppress the progress meter but will show error messages if the request fails.

Reply:
Thank you for the review - I updated the PR to use the `https://quay.io` as I feel it's simpler to just update the URL.